src/pages/Registration/Steps/getRegistrationState.ts

```diff
@@ -40,22 +40,23 @@ export function getRegistrationState(
   if (isDoneBanking(reg)) {
     const { ContactPerson: c, Registration: r } = reg;
     return {
-      step: 5,
+      step: 6,
```

Comment:
@ap-justin not sure if the `isSubmitted(reg)` check above is even necessary?

Reply:
Addressed here https://github.com/AngelProtocolFinance/angelprotocol-web-app/pull/2506#discussion_r1402000976